test/entrypoint/srv_select.lua

```diff
@@ -126,6 +126,40 @@ package.preload['customers-storage'] = function()
                 unique = false,
                 if_not_exists = true,
             })
+
+            local developers_space = box.schema.space.create('developers', {
+                format = {
+                    {name = 'id', type = 'unsigned'},
+                    {name = 'bucket_id', type = 'unsigned'},
+                    {name = 'name', type = 'string'},
+                    {name = 'last_name', type = 'string'},
+                    {name = 'age', type = 'number'},
+                    {name = 'additional', type = 'any'},
+                },
+                if_not_exists = true,
+                engine = engine,
+            })
+
+            -- primary index
+            developers_space:create_index('id_index', {
+                parts = { 'id' },
+                if_not_exists = true,
+            })
+
+            developers_space:create_index('bucket_id', {
+                parts = { 'bucket_id' },
+                unique = false,
+                if_not_exists = true,
+            })
+
+            developers_space:create_index('full_name', {
+                parts = {
+                    {field = 'name'},
+                    {field = 'last_name'},
+                },
+                unique = false,
+                if_not_exists = true,
+            })
```

Comment:
Where is at least one jsonpath index?

Reply:
I expect to see following tests:
  - primary key is jsonpath
  - secondary key is jsonpath
  - pagination for indexes above
  - sharding key calculation for such indexes (aka bucket optimization)